src/Ledger/Certs.lagda

```diff
@@ -429,7 +429,7 @@ data _⊢_⇀⦇_,CERTBASE⦈_ where
     let open PParams pp
         refresh         = mapPartial getDRepVote (fromList vs)
         refreshedDReps  = mapValueRestricted (const (e + drepActivity)) dreps refresh
-        wdrlCreds       = mapˢ stake (dom wdrls)
+        wdrlCreds       = filterˢ isKeyHash (mapˢ stake (dom wdrls))
```

Comment:
You shouldn't update the binding here, since `wdrlCreds` is used twice. But the only place where the filter should happen is in `wdrlCreds ⊆ dom voteDelegs`, not in the state update. Otherwise we'd break preservation of Ada.

Reply:
@WhatisRT This should be fixed now.